mysite/settings.py

```diff
@@ -115,7 +116,7 @@
 
 STATIC_URL = "/static/"
 
-django_heroku.settings(locals())
+# django_heroku.settings(locals())
```

Comment:
Same - why did you comment this out?

Reply:
This is was just to work locally. Forgot to uncomment. Thanks